Content.Server/Xenoarchaeology/XenoArtifacts/Effects/Systems/PolyArtifactSystem.cs

```diff
@@ -0,0 +1,39 @@
+using Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Components;
+using Content.Server.Xenoarchaeology.XenoArtifacts.Events;
+using Content.Shared.Humanoid;
+using Content.Server.Polymorph.Systems;
+using Content.Shared.Zombies;
+using Content.Shared.Mobs.Systems;
+
+namespace Content.Server.Xenoarchaeology.XenoArtifacts.Effects.Systems;
+
+public sealed class PolyArtifactSystem : EntitySystem
+{
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
+    [Dependency] private readonly MobStateSystem _mob = default!;
+    [Dependency] private readonly PolymorphSystem _poly = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    public override void Initialize()
+    {
+        SubscribeLocalEvent<PolyArtifactComponent, ArtifactActivatedEvent>(OnActivate);
+    }
+
+    /// <summary>
+    /// Provided target is alive and is not a zombie, polymorphs the target.
+    /// </summary>
```

Comment:
why the zombie check? feels jank

Reply:
so you dont polymorph a zombie, it kinda messes up when it tries to revert the player back to previous.

Reply:
this should still be using a different check, or remove the check and make an issue bc that just sounds like a bug

Reply:
going to leave zombiecomp check in, i think its better than a zombie getting all messed up when it turns the zombie into monkey then tries to revert it to zombie again.

Reply:
K i will test it more indepth, see eaxctly what weirds out. Just seeing a zombie turn into a non zombie monkey is a little worrying.

Reply:
It does work, zombie turns into monkey, in monkey form zombie acts like monkey and isnt a zombie but when timer runs out monkey reverts to zombie and will attack again and behave like zombie.